portality/crosswalks/article_crossref_xml.py

```diff
@@ -235,6 +235,8 @@ def crosswalk_article(self, record, journal):
 
         # authors
         contributors = record.find("x:contributors", NS)
+        if contributors is None:
+            raise CrosswalkException(message="No contributors found.", inner_message="DOAJ requires at least one author for each article")
```

Comment:
Could we place these bits of text in the `Messages` file.  Otherwise, that's fine with me